Toolset/palettes/inspector/editors/com.livecode.pi.textcontents.behavior.livecodescript

```diff
@@ -41,15 +41,28 @@ constant kMaxHeight = 800
 on editorResize
    lock screen
    lock messages
+   local tManualHeight, tManualGroupHeight
+   put getManualHeight() into tManualHeight
+   if tManualHeight <> empty then
+      put tManualHeight - the height of group "controls" of me - kControlGap - 1 - the height of group "background" into tManualGroupHeight
```

Comment:
Same question with the 1 here - just obtained through observation?

Reply:
I think is was needed to hide the vertical scrollbar. Obtained through observation